### DISCUSSION TITLE
Welcome to Jarvis-v2.0 Discussions!

### DISCUSSION THREAD

**Discussion by JoelShine** [JoelShine/Jarvis-v2.0#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JoelShine** [comment]
**Hi ! I am Joel Shine and this is my discussions group. You can give suggestions about my project or any add-ons to make this project wonderful.**

**Reply by Redfox1981** [reply]
hey im such a "nub" i know . with that said im not sure if your around to help but here goes . 
i know im doing the Jarvis v2.0 wrong and i dont want to give up and probly should have reached out to you a lot sooner . i love the idea of this program and the tweaks that i can apply to it . for some odd reason i can get it to run . ive copided all the asking files to nano like the script says to . ive tried and tried and ill keep trying but if you could reach out and point me in the right direction id really really appreciate it . this program seems to have the abilities to help me more than imagined and i do have a couple ideas to tweak the engine and want to also know how to go about it the right way this time cause ill jump at it from every angle and end up breaking my pc like i usually do . none the less you seem like a very good candidate to learn from so please help . good day . Pss EMAIL : 
redbloodedfox1@gmail.com 
my name is joseph by the way .

**Reply by Flashxyz** [reply]
Can I please have the full Python script because I'm running a Mac :):

my email:

omarfahmad2010@gmail.com

**Reply by Redfox1981** [reply]
hey big guy . im gonna give jarvis a try again . even went to the lengths to get another pc . eventually once i figure this out or when you get a chance to help speed things up a bit eithier waystill have yet to get this thing up and running .

**Comment by JairoJosy** [comment]
Hi, SIr
Can I please get the full python script, please ?
Here is my email:
jairojosym2020@gmail.com``````

**Reply by JoelShine** [reply]
Hello @JairoJosy , this is Joel Shine. I have attached the code for Jarvis in the mail which I sent in your email.
Also, please check out my latest jarvis program updated with even more features including offline speech recognition and very intelligent chat engine - https://github.com/JoelShine/JARVIS-AI-ASSISTANT

Please support by starring some of the repositories. All the code and steps are included in the repository itself. For further clarification, do mail me.

Thank you,
Regards,
Joel Shine

**Comment by Zubin87** [comment]
Hi, SIr
Can I please get the full python script, please ?
Here is my email:
zubinbhatt003@gmail.com

**Comment by yasir-shahhhh** [comment]
use llm api's to make responces better

**Comment by tiniestorpheus** [comment]
Hello can i have the frontend script containing the jarvis GIF

**Comment by kennethpont** [comment]
Just recently setup a LLM on a beefy linux system (LLMs are huge hogs of CPU and RAM) and saw this a a cool front end to send discussions through and add additional capabilities, seeing as your documentation says you have this mostly configured for running on a Windows system, what minimal hardware does something like this require?  Obvoiusly 10th gen or higher for Win11, but does it need something more capaable than your basic Intel GPU, more than your i5 or i7 and how much memory?
